### ISSUE TITLE
Error building on arm64

### ISSUE THREAD

**Issue by xshyne** [shouya/rss-funnel#43]
I'm trying to build the docker image on arm64 and i get an error with rquickjs

```
error: couldn't read /home/xshyne/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rquickjs-sys-0.4.0/src/bindings/aarch64-unknown-linux-gnu.rs: No such file or directory (os error 2)
  --> /home/xshyne/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rquickjs-sys-0.4.0/src/lib.rs:16:1
   |
16 | include!(concat!("bindings/", bindings_env!("TARGET"), ".rs"));
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `include` (in Nightly builds, run with -Z macro-backtrace for more info)

The following warnings were emitted during compilation:

warning: rquickjs probably doesn't ship bindings for platform `aarch64-unknown-linux-gnu`. try the `bindgen` feature instead.

error: could not compile `rquickjs-sys` (lib) due to previous error
```

It seems that with the version 0.4.0 of rquickjs is not possible to build on arm64. It's fixed in the version 0.5.0.
The file aarch64-unknown-linux-gnu.rs got deleted during 0.4.0 beta and was added back for the 0.4.3.

**Comment by shouya** [comment]
Thanks for reporting. I'll look into this. Previously, I had success building for `aarch64-unknown-linux-musl` target (see Makefile), so you can try it if you want.

**Comment by xshyne** [comment]
I tried a little more and the problem seems to be only for `aarch64-unknown-linux-gnu` target. I compiled for musl and it works without issue. (i didn't really knew the difference between gnu and musl)
So this issue is just if you want to support compilation for gnu.

**Comment by shouya** [comment]
The main difference is that `-gnu` libc is dynamically linked and musl is statically linked.

```
$ ldd target/aarch64-unknown-linux-musl/release/rss-funnel
	not a dynamic executable
```

I chose to release in `-musl` target because I want a minimal binary without any runtime dependency, so the binary can run in https://hub.docker.com/_/scratch/.

That said, it's still keeping up to the latest rquickjs and make it possible to build `-gnu` if the user wants it. I'll leave this issue open until I upgraded the dependency.
